yt_dlp/extractor/youtube.py

```diff
@@ -567,9 +566,15 @@ def _initialize_pref(self):
         pref.update({'hl': self._preferred_lang or 'en', 'tz': 'UTC'})
         self._set_cookie('.youtube.com', name='PREF', value=urllib.parse.urlencode(pref))
 
+    def _initialize_cookie_auth(self):
+        self._load_sid_cookies()
+        if self._SAPISID or self._1PSAPISID or self._3PSAPISID:
+            self.write_debug('Logged in using cookies')
+
     def _real_initialize(self):
         self._initialize_pref()
         self._initialize_consent()
+        self._initialize_cookie_auth()
```

Comment:
This change will result in "Logged in using cookies" being output when the extractor is initialized, which could be misleading if the cookies have been rotated/invalidated. Previously, the debug message wasn't triggered until after the initial webpage request, at which point the cookies would have been discarded if they were no longer valid

Reply:
Perhaps we could change it to something like "Found YouTube cookies"? We don't care if they are valid or not, just that we have detected them (for debugging purposes)

Reply:
Updated